src/main.cpp

```diff
@@ -557,14 +529,28 @@ int main ( int argc, char** argv )
             continue;
         }
 
+        // Enable Special Options ----------------------------------------------
+        if ( cmdLine.GetFlagArgument ( i, CMDLN_SPECIAL ) )
+        {
+            bSpecialOptions = true;
+            qInfo() << "- Special options enabled !";
+            continue;
+        }
+
         // Unknown option ------------------------------------------------------
-        qCritical() << qUtf8Printable ( QString ( "%1: Unknown option '%2' -- use '--help' for help" ).arg ( argv[0] ).arg ( argv[i] ) );
+        // No exit for options after the "--special" option.
+        // Used for debugging and testing new options...
+        if ( !bSpecialOptions )
+        {
+            // Unknown option ------------------------------------------------------
+            qCritical() << qUtf8Printable ( QString ( "%1: Unknown option '%2' -- use '--help' for help" ).arg ( argv[0] ).arg ( argv[i] ) );
 
 // clicking on the Mac application bundle, the actual application
 // is called with weird command line args -> do not exit on these
```

Comment:
If this is true, how about all other places in main using almost the same code ?

```
qCritical() << qUtf8Printable ( QString (...) );
exit (1);
```
OnFatalError uses this code now too.